__snapshots__/client/components/Sidebar/Sidebar.test.tsx.snap

```diff
@@ -2,41 +2,41 @@
 
 exports[`Test Sidebar Sidebar renders correctly 1`] = `
 <div
-  className="sc-feJyhm hgExDA sidebar"
+  className="sc-iELTvK hWGXaO sidebar"
```

Comment:
why's this changed

Reply:
I don't really know...but does it really matter?